### ISSUE TITLE
How to cross compile Node.js for MIPS on a X86 system

### ISSUE THREAD

**Issue by unional** [v8mips/v8mips#492]
I want to run node.js on mips. However my mips system does not have the capability to build.

What are the steps to do a cross-compiling of node.js on x86 machine?

I'm able to build mip64 on the x86 machine based on this instruction: https://github.com/v8mips/v8mips/wiki/Building-v8-for-MIPS#cross-compiling-for-a-mips64-target
(abid with a few warnings as mentioned in #491)

The section in wiki about this is empty right now:
https://github.com/v8mips/v8mips/wiki/Node.js-with-mips-v8#cross-compiling

**Comment by mkaric** [comment]
Yes, we have not updated that information on wiki in a while. Maybe node repository on github is a better source of information on building node.js. I found this information which may be useful for you:
https://github.com/nodejs/node/issues/8210.

**Comment by unional** [comment]
Many thanks! 🎉  I'll try it out and if that works I'll help update the Wiki.

**Comment by mkaric** [comment]
Thank you, let us know if it works.

**Comment by unional** [comment]
Just an update, I'm running into some problems and trying to get that resolved: https://github.com/nodejs/node/issues/13488
